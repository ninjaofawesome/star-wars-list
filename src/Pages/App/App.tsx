import React from 'react';

// components
import Table from "../../Components/Table/Table";
import './App.css';

const App: React.FC = () => (
	<div className="App">
		<Table />
	</div>
);

export default App;
