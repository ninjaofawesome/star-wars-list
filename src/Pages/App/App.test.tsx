import React from 'react';
import { shallow } from 'enzyme';
import App from './App';

describe('App Page', () => {
	test('renders the page', () => {
		const app = shallow(<App />);
		expect(app).toMatchSnapshot();
	}); 
});
