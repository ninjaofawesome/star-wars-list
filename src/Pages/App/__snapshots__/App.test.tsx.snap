// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App Page renders the page 1`] = `ShallowWrapper {}`;
